### PR TITLE
Review D0500 Slides for activities in day 1

### PR DESCRIPTION
**Pull request by rmarting**
This PR includes the following main changes to add into master repository:

:memo: Used Markdown/Asciidoc best practices in README files
:art: Fixed some typos in slides (indentation, dots, ...) 
:art: Added or updated OPL links

### CODE REVIEW

**Comment by rmarting** [comment]
Is there some ETA to review and approve this PR? We are preparing an enablement and it should be nice to have this content added.

File: slides/markdown/kick_off.md [hunk 1]
```diff
@@ -26,9 +26,15 @@ DO500 <!-- {_class="title-color"} -->
 
 
 <!-- .slide: data-background-image="images/chef-background.png" class="white-style" -->
-### DevOps practices used in this section:
+### DevOps practices introduced in this section:
```
Comment (jtudelag) [inline slides/markdown/kick_off.md:29]: @rmarting I am not a native speaker...so not sure what is best, "used" or "introduced".  I would rather keep the "used", at least for consistency with the rest of the content.

File: slides/markdown/team_identity.md [hunk 2]
```diff
@@ -30,4 +30,4 @@ Form a straight line across the room in order of relative technical experience i
 
 <!-- .slide: data-background-image="images/chef-background.png", class="white-style" -->
 ### DevOps practices used in this section:
-- Team Identity
```
Comment (jtudelag) [inline slides/markdown/team_identity.md:33]: @rmarting can you revert this change? The practice is in fact "Team Identity". "Team Sentiment" is a a different practice. On the other hand, "Team Identity" is not describe in the OPL yet, so I opened an issue in the OPL repo to add it: https://github.com/openpracticelibrary/openpracticelibrary/issues/1440#issuecomment-796767009

**Review by jtudelag — COMMENTED** [review]
@rmarting can you check my comments? Also this would need to be rebased, there are conflicts with master.  Thanks!

File: slides/markdown/kick_off.md (hunk 1 above)
Comment (rmarting) [reply slides/markdown/kick_off.md]: Thanks @jtudelag ! It makes sense your suggestion. Applied

File: slides/markdown/team_identity.md (hunk 2 above)
Comment (rmarting) [reply slides/markdown/team_identity.md:33]: Removed the OPL practice. Waiting to have the new one to link it.

**Comment by rmarting** [comment]
@jtudelag Do you need anything else in this PR review? Thanks a lot for your time.

**Comment by rmarting** [comment]
@jtudelag could you review this latest PR?

**Comment by jtudelag** [comment]
LGTM!
